src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+
+  return (num1) => {
+    return (num2) => {
```

Comment:
```suggestion
    return (secondNumber) => {
```
не використовуй цифри в назвах змінних